Sources/Networking/HTTPClient/HTTPRequest.swift

```diff
@@ -140,7 +142,7 @@ extension HTTPRequest.Path {
                 .postSubscriberAttributes,
                 .postAdServicesToken:
             return true
-        case .health:
```

Comment:
I guess this is because if this is going to live in a different server, we might not support etags in that place for now?

Reply:
Exactly.